database/models.py

```diff
@@ -236,3 +233,29 @@ class Document(VersionedBase):
     decision: Mapped[bool]
     decision_date: Mapped[Optional[timestamp]]
     # file
+
+
+class LifeCycleDate(VersionedBase):
+    """
+    Elinkaaritilan päivämäärät
+    """
+
+    __tablename__ = "lifecycle_date"
+
+    lifecycle_status_id: Mapped[uuid.UUID] = mapped_column(
+        ForeignKey("codes.lifecycle_status.id", name="plan_lifecycle_status_id_fkey"),
+        index=True,
+    )
+    plan_id: Mapped[uuid.UUID] = mapped_column(
```

Comment:
I guess the foreign keys must be nullable, because we don't want the same object pointing to plan, plan_regulation and plan_proposition all at once. So let's have `Mapped[Optional[uuid.UUID]]` here.